processor/stream/benchmark_test.go

```diff
@@ -64,15 +66,15 @@ func BenchmarkStreamProcessor(b *testing.B) {
 				b.StopTimer()
 				r.Reset(data)
 				b.StartTimer()
-				sp.HandleStream(context.Background(), rl, map[string]interface{}{}, r, report)
+				processor.HandleStream(context.Background(), rl, map[string]interface{}{}, r, report)
 			}
 		}
 	}
 
 	for _, f := range files {
-		b.Run(f.Name(), func(b *testing.B) {
-			b.Run("NoRateLimit", benchmark(f.Name(), nil))
-			b.Run("WithRateLimit", benchmark(f.Name(), rl))
+		b.Run(filepath.Base(f), func(b *testing.B) {
+			b.Run("NoRateLimit", benchmark(f, nil))
+			b.Run("WithRateLimit", benchmark(f, rl))
```

Comment:
There is no rate limiter used for backend processors, therefore the comparison is only really necessary for RUM, but it's fine to have it run for both.

Reply:
The test could probably do with further refactoring, but I'll leave that for another day. The `BackendProcessor` is currently also processing v2 RUM events - that's unchanged from before.